src/ResourceManager/IotCentral/Commands.IotCentral/IotCentral/GetAzureRmIotCentralApp.cs

```diff
@@ -0,0 +1,89 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.IotCentral.Common;
+using Microsoft.Azure.Commands.IotCentral.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.IotCentral;
+using Microsoft.Azure.Management.IotCentral.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.IotCentral
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "IotCentralApp", DefaultParameterSetName = ListIotCentralAppsParameterSet)]
+    [OutputType(typeof(PSIotCentralApp))]
+    public class GetAzureRmIotCentralApp : IotCentralBaseCmdlet
+    {
+        const string ListIotCentralAppsParameterSet = "ListIotCentralAppsParameterSet";
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
@cle5eland same comment about removing the `ValueFromPipelineByPropertyName` property from this parameter (in both of the parameter attributes for `-ResourceGroupName`)

Reply:
Fixed.